From 578b244b788cde45ab4b413106bd7c04b9610dda Mon Sep 17 00:00:00 2001
From: bryanloz <bryanloz@xilinx.com>
Date: Sat, 2 Jul 2022 14:42:55 -0600
Subject: [PATCH 03/15] prid64 macro usage

---
 src/CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 6deda2c42..104e0fa27 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -9,6 +9,8 @@ ENABLE_TESTING()
 
 SET(PROJECT_DESCRIPTION "https://github.com/Xilinx/XRT")
 
+add_compile_options("-D__STDC_FORMAT_MACROS=1")
+
 # Include supporting CMake functions
 include(CMake/unitTestSupport.cmake)
 
