--- orig/src/runtime_src/xdp/profile/writer/vp_base/vp_writer.h	2022-04-14 15:29:02
+++ work/src/runtime_src/xdp/profile/writer/vp_base/vp_writer.h	2025-12-10 18:09:35
@@ -19,6 +19,7 @@
 
 #include <fstream>
 #include <string>
+#include <cstdint>
 
 #include "xdp/config.h"
 
@@ -28,9 +29,9 @@
   class VPDatabase ;
   class DeviceIntf ;
 
-  // The base class for all writers, including summaries, traces, 
+  // The base class for all writers, including summaries, traces,
   //  and any others.
-  class VPWriter 
+  class VPWriter
   {
   private:
     // The base name of all files created by this writer
@@ -57,7 +58,7 @@
 
     VPWriter() = delete ;
 
-    inline const char* getRawBasename() { return basename.c_str() ; } 
+    inline const char* getRawBasename() { return basename.c_str() ; }
     XDP_EXPORT virtual void switchFiles() ;
     XDP_EXPORT virtual void refreshFile() ;
   public:
@@ -71,8 +72,8 @@
     // Return false to indicate no data was written
     virtual bool write(bool openNewFile = true) = 0 ;
     virtual bool write(bool /*openNewFile*/, void* /*handle*/) {return false;}
-    virtual bool isDeviceWriter() { return false ; } 
-    virtual DeviceIntf* device() { return nullptr ; } 
+    virtual bool isDeviceWriter() { return false ; }
+    virtual DeviceIntf* device() { return nullptr ; }
     virtual bool isSameDevice(void* /*handle*/) { return false ; }
 
     virtual std::string getDirectory() { return directory ; }
